pom.xml

```diff
@@ -27,6 +27,13 @@
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.jsoup</groupId>
+			<artifactId>jsoup</artifactId>
+			<version>1.13.1</version>
+		</dependency>
```

Comment:
Is this dependency necessary anymore?

Reply:
Thanks for the suggestion, removed the code now